src/components/BookItem.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function BookItem({ book }) {
+  const { title, author } = book;
+  return (
+    <div>
+      <h2>
+        {title}
+      </h2>
+      <h2>
+        {author}
+      </h2>
+    </div>
```

Comment:
- [ ] This component should contain a button to remove the correspondent book from the list of books

Reply:
The button that should be visible at this stage is only the ADD BUTTON, which I have showcased

Reply:
@Moses-chibuike, it's clear in the requirement that the book component should have a remove button 👇🏼 

![Screenshot_20230331_031120](https://user-images.githubusercontent.com/25227022/229013437-90356c6f-efab-4905-8781-152b24a22902.png)